src/lib/styles/themes/theme-core.scss

```diff
@@ -86,7 +86,7 @@ body
         
         //backgrounds and structural elements
         --color-structure-0: #FFFFFF; //0 white
-        --color-structure-1: #F5F5F5; //10
+        --color-structure-1: #F3F4F5; //10
```

Comment:
ist es wirklich so gewollt, eine Farbe an über hundert Stellen in Terra zu verändern?

Reply:
das ist immer noch grau, ist von Saman geändert worden

Die checkboxen sind ja auch noch nicht umgestylt wurden

Reply:
Ja das ist so gewollt. Das ist eine sehr kleine Veränderung des grau-Wertes. Es war vorher leicht rötlich und jetzt geht es leicht ins bläuliche. An der grau-Stärke hat sich nichts geändert. Es sieht jetzt etwas harmonischer aus mit dem Rest des Backends. Für die meisten wahrscheinlich gar nicht wahrnehmbar.